htdocs/js/jquery.quickreply.js

```diff
@@ -156,23 +159,19 @@ jQuery(function($) {
         }
     }
 
-    $("#submitpview").live("click", function(e){
+    $("#submitpview").on("click", function(e){
         $("#qrform input[name='submitpreview']").val(1);
         submitform(e);
     });
-    $("#submitpost").live("click", function(e){
-        var maxlength = 16000;
-        var length = $("#body").val().length;
-        if ( length > maxlength ) {
-            alert("Sorry, but your comment of " + length + " characters exceeds the maximum character length of " + maxlength + ". Please try shortening it and then post again");
-
+    $("#submitpost").on("click", function(e){
+        if ( checkCommentLength() ) {
+            submitform(e);
+        } else {
             e.stopPropagation();
             e.preventDefault();
-        } else {
-            submitform(e);
         }
     });
-    $("#submitmoreopts").live("click", function(e) {
+    $("#submitmoreopts").on("click", function(e) {
```

Comment:
Oh, I should explain this. “Live” means it keeps an eye out for any new instances of the desired selection that get created for the lifetime of the page, and attach the listener to them as well. But we don’t spawn new reply forms willy nilly. 

Remember how the qrform used to get appended to the page with mid-stream js? It was probably related to that. That, or just paranoia maybe.